src/packages/@ncigdc/modern_components/ClinicalAnalysis/ContinuousAggregationQuery.js

```diff
@@ -27,18 +27,21 @@ const getContinuousAggs = ({ fieldName, stats, filters, bins }) => {
   }
 
   let rangeArr = _.reduce(bins, (acc, bin, key) => {
+    const binValues = bin.key.split('-').map(keyValue => Number(keyValue));
+    const binFrom = binValues[0];
+    const binTo = binValues[1];
     if (
       !!bin &&
```

Comment:
get correct values from the bin

allow buckets that fall outside of available data